src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -626,7 +626,7 @@
                             </MenuItem>
                             <MenuItem x:Name="OnBoardingMenuGuide"
                                       Header="{x:Static p:Resources.OnboardingGuide}"
-                                      Click="OnBoardingMenuGuide_Click">
+                                      Command="{Binding GettingStartedGuideCommand}">
```

Comment:
@QilongTang 
The same fix of using a command will need to be applied to the Packages guide? including all the stuff related to ASM? or you are considering a new task?

Reply:
Hi @RobertGlobant20 Only the second tour is depending on ASM loaded so no plan for that change